content/applications/finance/expenses.rst

```diff
@@ -4,231 +4,544 @@
+smart button.
+
+.. image:: expenses/receipt-smartbutton.png
+   :align: center
+   :alt: Attach a receipt after saving the record.
+
+Automatically create new expenses from an email
+-----------------------------------------------
+
+Instead of individually creating each expense in the *Expenses* app, expenses can be automatically
+created by sending an email to an email alias.
+
+To do so, first, an email alias needs to be configured. Go to :menuselection:`Expenses app -->
+Configuration --> Settings`. Ensure :guilabel:`Incoming Emails` is checked off.
+
+.. image:: expenses/email-alias.png
+   :align: center
+   :alt: Create the domain alias by clicking the link.
+
+.. note::
+   If the domain alias needs to be set up, :guilabel:`Setup your domain alias` will appear beneath
+   the incoming emails check box instead of the email address field. If the domain alias needs to be
+   setup, refer to this documentation: :doc:`/administration/maintain/domain_names`. Once the domain
+   alias is configured, the email address field will be visible beneath the incoming emails section.
+
+Next, enter the email address to be used in the email field, then click :guilabel:`Save`. Now that
+the email address has been entered, emails can be sent to create new expenses without having to be
+in the Odoo database.
+
+To submit an expense via email, create a new email and enter the product's *internal reference* code
+(if available) and the amount of the expense in the email subject. Next, attach the receipt to the
+email. Odoo creates the expense by taking the information in the email subject and combining it with
+the receipt.
+
+To check an expense product's internal reference, go to :menuselection:`Expenses app -->
+Configuration --> Expense Products`. If an internal reference is listed on the product, it is
+visible in this view as :guilabel:`(Ref###)`.
+
+.. image:: expenses/internal-ref-numbers.png
+   :align: center
+   :alt: Internal reference numbers are listed in the main Expense Products view.
+
+To add an internal reference on an expense product, click on the product, then click
+:guilabel:`Edit`. In edit mode, enter the :guilabel:`Internal Reference` in the field. Beneath the
+:guilabel:`Internal Reference` field, this sentence appears: :guilabel:`Use this reference as a
+subject prefix when submitting by email.`.
+
+.. image:: expenses/meals.png
+   :align: center
+   :alt: Internal reference numbers are listed in the main Expense Products view.
+
+.. note::
+   For security purposes, only authenticated employee emails are accepted by Odoo when creating an
+   expense from an email. To confirm an authenticated employee email address, go to the employee
+   card in the :guilabel:`Employees` app, and refer to the :guilabel:`Work Email` in the main field.
+
+   .. image:: expenses/authenticated-email-address.png
+      :align: center
+      :alt: Create the domain alias by clicking the link.
+
+.. example::
```

Comment:
Because of the edits, my numbers in the RST doc are a mess and I am not sure which line is 196 - and trying to move it to a 'better' spot, I think the only other spot would be after the "next, enter the email address" paragraph. But since I also have the 'set up' called out in the image, I think it is still good to keep it here, since it matches the image.